x-pack/plugins/cases/public/common/mock/test_providers.tsx

```diff
@@ -47,6 +54,43 @@ TestProvidersComponent.displayName = 'TestProviders';
 
 export const TestProviders = React.memo(TestProvidersComponent);
 
+export interface AppMockRenderer {
+  render: UiRender;
+  coreStart: StartServices;
+}
+
+export const createAppMockRenderer = ({
+  features,
+  owner = [SECURITY_SOLUTION_OWNER],
+  userCanCrud = true,
+}: {
+  features?: CasesFeatures;
+  owner?: string[];
+  userCanCrud?: boolean;
+} = {}): AppMockRenderer => {
+  const services = createStartServicesMock();
+
+  const AppWrapper: React.FC<{ children: React.ReactElement }> = ({ children }) => (
+    <I18nProvider>
+      <KibanaContextProvider services={services}>
+        <ThemeProvider theme={() => ({ eui: euiDarkVars, darkMode: true })}>
+          <CasesProvider value={{ features, owner, userCanCrud }}>{children}</CasesProvider>
+        </ThemeProvider>
+      </KibanaContextProvider>
+    </I18nProvider>
+  );
+  const render: UiRender = (ui, options) => {
+    return reactRender(ui, {
+      wrapper: AppWrapper as React.ComponentType,
+      ...options,
+    });
+  };
+  return {
+    coreStart: services,
```

Comment:
What do you think about changing it to `mocks: { coreStart: services, ...someOtherMocks }`? I think it makes it clearer of what the `coreStart` is.

Reply:
considering the whole returned object is a mock I'd say it would be redundant.